apps/server/src/institutions/getInstitutionHandler.ts

```diff
@@ -6,7 +6,8 @@ export const getInstitutionHandler = async (req: Request, res: Response) => {
   const ucpInstitutionId = req.params.institution_guid;
 
   const resolvedInstitution = await resolveInstitutionAggregator({
-    aggregatorOverride: req.context.aggregator,
+    aggregatorOverride:
+      req.context.aggregatorOverride || req.context.aggregator,
```

Comment:
Could you add a test to getInstitutionHandler.test.ts for this change. It will probably be a duplicate of another test, just with a different variable in the context